src/Infrastructure/LeanCode.Kratos/KratosAuthenticationHandler.cs

```diff
@@ -76,49 +76,49 @@ protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
         }
     }
 
-    protected virtual Task<KratosSession?> GetSessionAsync()
+    protected virtual async Task<IToSessionApiResponse?> GetSessionAsync()
```

Comment:
The generated result type is now an interface that also exposes some additional info about whether the call succeeded or not.